spack-repo/packages/singularity-eos/package.py

```diff
@@ -63,6 +81,19 @@ class SingularityEos(CMakePackage, CudaPackage):
 
     variant("closure", default=True, description="Build closure module")
 
+    plugins = {}
+
+    singularity_eos_plugin("dust", "example/plugin")
+
+    variant(
+        "plugins",
+        multi=True,
+        default="none",
+        validator=plugin_validator,
+        description="list of plugins to build",
+    )
+    variant("variant", default="default", description="include path used for variant header")
```

Comment:
Should these be version-gated on `develop` until a new release? I.e., so someone doesn't use this spackage to download a version of the source code with the missing cmake?

Reply:
Yes, good catch.